backend/user/src/main/kotlin/myapp/plugins/HTTP.kt

```diff
@@ -23,6 +23,8 @@ fun Application.configureHTTP() {
         allowHeader(HttpHeaders.ContentType)
         allowHeader(HttpHeaders.AccessControlAllowHeaders)
         allowHeader(HttpHeaders.AccessControlAllowCredentials)
-        anyHost()
+        // FIXME: 環境変数から取るようにしたほうがよさそう
+        allowHost("localhost:3000")
```

Comment:
環境変数使えばdockerで書くなりenvに書くなり好き勝手できるので、とりあえず環境変数を読み取る形にするのが一般的ですね〜
```suggestion
        allowHost(System.getenv("ALLOW_HOST") ?: "localhost:3000")
```

Reply:
allowHost(System.getenv("ALLOW_HOST") ?: "localhost:3000")